dispatch/dispatch_types.go

```diff
@@ -9,7 +9,7 @@ import (
 
 const (
 	// DefaultJobBuffer defines a maxiumum amount of jobs allowed in channel
-	DefaultJobBuffer = 100
+	DefaultJobBuffer = 1000
```

Comment:
I am having no issues with reverting this one back on linux.

Reply:
Yeah I'll be making this a config var so people can change it permanently if they enable many many currencies